_posts/0000-01-02-Boyce-Bellair.md

```diff
@@ -0,0 +1,6 @@
+--
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![A TEAM IMAGE](https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages2.minutemediacdn.com%2Fimage%2Fupload%2Fc_crop%2Ch_358%2Cw_640%2Cx_0%2Cy_12%2Ff_auto%2Cq_auto%2Cw_1100%2Fv1555924029%2Fshape%2Fmentalfloss%2Fa-team_1.jpg&imgrefurl=https%3A%2F%2Fwww.mentalfloss.com%2Farticle%2F55922%2Ffacts-about-the-a-team&tbnid=oeEZrfE0ycGBuM&vet=12ahUKEwj4q4rdmorqAhVFmp4KHdLFD8sQMygBegUIARDSAQ..i&docid=FRcfPo68TgDLlM&w=1100&h=615&q=THE%20A%20TEAM&ved=2ahUKEwj4q4rdmorqAhVFmp4KHdLFD8sQMygBegUIARDSAQ)